src/center.css

```diff
@@ -0,0 +1,30 @@
+
+.center{
+    display: flex;
+    transform: translate(0,+50%);
+
+  }
+
+.resized{
+    width: 400px;
+    height: 200px;
+}
+
+.top{
+  float:top;
+}
+
+.down{
+  float:bottom;
+}
+
+.grid{
+    grid-template-rows: repeat(1,auto);
+    grid-template-columns: 500px 200px;
```

Comment:
If you want to have responsive design, you probably should not use the `px` unit here. Please take a look at [this](https://stackoverflow.com/a/12271250/9335767) (especially the first link from the answer).